views/components/Graphs/BarGraph.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import { 
+  VictoryBar, 
+  VictoryChart, 
+  VictoryAxis, 
+  VictoryTheme, 
+  VictoryStack } from 'victory';
+
+
+const BarGraph = ({ graph }) => {
+  const tickValues = graph.data.map((_, i) => (i))
```

Comment:
what are the `ticketValues`? It seems like this will be an array of index values and I'm trying to understand how that's helping.

Reply:
[a2afc3](https://github.com/AMP-SCZ/dpdash/pull/91/commits/a2afc3e7f7b1f052bfef361f91e567c91849655a)

Reply:
I was using the tick values to make the percentages but it seems that going for a count on the y axis is better